build.gradle

```diff
@@ -1,20 +1,28 @@
 apply plugin: 'java'
 
+sourceCompatibility = '1.8'
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '4.5.1'
+    distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
+}
+
 // Dependency repository.
 repositories {
     mavenCentral()
 }
 
 // Project dependencies. Currently only for unit tests.
 dependencies {
-    testCompile 'junit:junit:4.12',
-                'com.google.code.gson:gson:2.8.0'
+    testCompile 'junit:junit:4.12';
+    testCompile 'com.google.code.gson:gson:2.8.0'
 }
 
 // Specify main class for the JAR file.
 jar {
     manifest {
-        attributes 'Main-Class': 'util.CommandLine'
+        attributes 'Main-Class': 'at.unisalzburg.apted.util.CommandLine'
```

Comment:
I post it here, because it's the first occurrence of that change.
 
Since it's the second pull request about packaging, I've decided to go for it. There are however two issues.
(1) Will such change affect the current users of the library? If yes, what are the options to deal with it?
(2) I'd prefer 'at.unisalzburg.dbresearch' instead of 'at.unisalzburg' prefix.

Reply:
Sure it will affect current users. Changing the package name is a breaking API change. On the other hand, the current situation is causing similar problems. To simplify the migration you could ship a version that contains both, the old packages and the new ones. In the old, you add a warning that will tell users that they have to migrate to the new package name.